dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/expression/PredictorExpression.java

```diff
@@ -166,6 +155,19 @@ public PredictorExpression( String expression )
                 "UID '%s' must start with a letter and contain 10 more letters and numbers in '%s'", degUid,
                 expression ) );
         }
+
+        // -->
```

Comment:
Perhaps remove this comment? Looks good otherwise!

Reply:
@larshelge, thanks. This is a comment for one of the six expression parts. The fifth part (array index 4) is the separator '-->'. I've now numbered the comments for each part and said more about this comment so hopefully that makes it clearer.